hack/build-images.sh

```diff
@@ -8,46 +8,14 @@
 # origin-deployer, etc.
 STARTTIME=$(date +%s)
 source "$(dirname "${BASH_SOURCE}")/lib/init.sh"
-source "${OS_ROOT}/contrib/node/install-sdn.sh"
-
-if [[ "${OS_RELEASE:-}" == "n" ]]; then
```

Comment:
We need to be clear to DevEx that the UX for this is changing, e-mail should be fine (along with the good error message in here if RPMs have not yet been built)